onyo/utils.py

```diff
@@ -79,8 +79,21 @@ def get_full_filepath(git_directory, file):
 
 def get_editor():
     editor = os.environ.get('EDITOR')
-    if editor is None:
-        editor = 'nano'
+    if (editor is None) or (editor == "") or (run_cmd("which " + editor).rstrip("\n") == ""):
```

Comment:
Two comments here:

This check for two different things:
1) if no EDITOR is specified
2) if the specified editor cannot be found in the path

The first can be simplified to use truthiness:
```
if not editor:
```

The second check should be its own if-check (before this one) and throw a warning. Then proceeding to the fallbacks makes sense.

Reply:
I included both changes. Thanks!